From 3653e1dcd2079a9e020c004c94ddd8dd559e42d9 Mon Sep 17 00:00:00 2001
Message-Id: <3653e1dcd2079a9e020c004c94ddd8dd559e42d9.1367947969.git.minovotn@redhat.com>
In-Reply-To: <707b9b97153063374d2530e72c49b1499fc21af9.1367947969.git.minovotn@redhat.com>
References: <707b9b97153063374d2530e72c49b1499fc21af9.1367947969.git.minovotn@redhat.com>
From: Laszlo Ersek <lersek@redhat.com>
Date: Mon, 6 May 2013 19:27:31 +0200
Subject: [PATCH 066/114] qapi: Fix enumeration typo error

RH-Author: Laszlo Ersek <lersek@redhat.com>
Message-id: <1367868499-27603-9-git-send-email-lersek@redhat.com>
Patchwork-id: 51107
O-Subject: [RHEL-6.5 qemu-kvm PATCH v2 08/56] qapi: Fix enumeration typo error
Bugzilla: 952873
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: Lei Li <lilei@linux.vnet.ibm.com>
Reviewed-by: Luiz Capitulino <lcapitulino@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@gmail.com>
(cherry picked from commit 6932a69b20a88428c531805cdd20eec8acf05b27)

Conflicts:

	qapi-schema.json
Signed-off-by: Laszlo Ersek <lersek@redhat.com>
---
 qapi-schema-guest.json |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qapi-schema-guest.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qapi-schema-guest.json b/qapi-schema-guest.json
index d955cf1..ed0eb69 100644
--- a/qapi-schema-guest.json
+++ b/qapi-schema-guest.json
@@ -293,7 +293,7 @@
 ##
 # @GuestFsFreezeStatus
 #
-# An enumation of filesystem freeze states
+# An enumeration of filesystem freeze states
 #
 # @thawed: filesystems thawed/unfrozen
 #
-- 
1.7.11.7

